vue/components/ui/atoms/button-icon/button-icon.vue

```diff
@@ -254,6 +258,18 @@ export const ButtonIcon = {
             };
             if (this.color) base["button-icon-" + this.color] = this.color;
             return base;
+        },
+        loaderStyle() {
+            const scaledSize = (this.size * 10) / 30;

```

Comment:
`/ 3` instead?

Reply:
Yup, I guess I just tried to maintain the reference to the values that I used as reference :P